src/coreclr/inc/bbsweep.h

```diff
@@ -299,7 +297,6 @@ class BBSweep
             if (pSD) delete [] ((char *) pSD);
             if (pAdminSid) FreeSid(pAdminSid);
             if (hToken) CloseHandle(hToken);
-            if (pACL) LocalFree(pACL);
```

Comment:
It looks like this would cause a memory leak - the `SetEntriesInAcl` above creates a new ACL that needs to be freed using LocalFree.
@jkotas I wonder though if we should get rid of the stuff under !FEATURE_CORESYSTEM instead.

Reply:
Yes, I noted it above (https://github.com/dotnet/runtime/pull/63954#discussion_r787376907) #63850 is deleting this whole block.

Reply:
Ah nice, I've missed that.